src/integration/resources/inprocess/dbnode_test.go

```diff
@@ -206,3 +211,23 @@ const defaultDBNodeConfig = `
 db:
   writeNewSeriesAsync: false
 `
+
+func TestLoad(t *testing.T) {
+	//amainsd delete
+	yamlCfg := `
+"db":
+  discovery:
+   "config":
+   "service":
+     "etcdClusters":
+     - "endpoints": ["http://127.0.0.1:2379"]
+       "zone": "embedded"
+     "service": "m3db"
+     "zone": "embedded"
+     "env": "default_env"
+`
+	var dbCfg dbcfg.Configuration
+	require.NoError(t, yaml.Unmarshal([]byte(yamlCfg), &dbCfg))
+	spew.Dump(dbCfg)
+	//fmt.Printf("%+v\n", dbCfg)
```

Comment:
Did you want to keep these debugs?

Reply:
Gah sorry, I should've taken a pass on these. Cleaning.